src/alacris_lib/side_chain.ml

```diff
@@ -338,6 +338,21 @@ module OperatorFeeSchedule = struct
   include (Persistable (PrePersistable) : PersistableS with type t := t)
 end
 
+(** What we need for the system is the pair of Revision and state digest *)
+module PairRevisionDigest = struct
```

Comment:
It is because the system works always with (confirmed_state, confirmed_revision).
Thus when we submit a claim we always need to have the pair. That's why I think it
is better to have this pairs being used in the system.
But maybe I can downgrade to just a type.

Reply:
If you think a module is warranted, I trust you—but module or type, it needs a better name than `Pair...` otherwise it doesn't even deserve a name, type or module.

`UpdateSummary` ? `StateRoot` ? `StateUpdate` ? `StateSummary` ? Something like that.

Reply:
Ok, done to **StateUpdate**.